apis/internal/query.d.ts

```diff
@@ -96,6 +96,15 @@ type Expressions<L,E> = KVPairs<L, Expression<Exclude<keyof E, symbol>>, ColumnV
     ? L
     : never
 
+type DeepPartial<T> = T extends object
```

Comment:
```suggestion
/**
  * @beta helper
  */
type DeepPartial<T> = T extends object
```
to show this is not part of the cds API